src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -28,7 +28,7 @@ function WebGLShadowMap( _renderer, _objects, _capabilities ) {
 
 		_maxTextureSize = _capabilities.maxTextureSize;
 
-	const shadowSide = { 0: BackSide, 1: FrontSide, 2: DoubleSide };
+	const shadowSide = { 0: BackSide, 1: FrontSide, 2: DoubleSide, 3: DoubleSide };
```

Comment:
Shouldn't we do `{ [ FrontSide ]: BackSide, [ BackSide ]: FrontSide, ... }` here?